setup.py

```diff
@@ -129,7 +130,15 @@ def finalize_options(self):
 
 ext_modules = []
 
-if "clean" not in sys.argv and "sdist" not in sys.argv:
+if "clean" in sys.argv:
+    # delete any previously Cythonized or compiled files in pygeos
+    p = Path('pygeos')
+    for pattern in ['*.c','*.so', '*.pyd']:
+        for filename in p.glob(pattern):
+            print("removing '{}'".format(filename))
```

Comment:
I added a print here, is that OK with you? 
That gives a visual indication that actually something is happening, which can be reassuring about what the command does.